soc/arm/nuvoton_npcx/common/soc_pins.h

```diff
@@ -59,6 +59,14 @@ struct npcx_alt {
 void soc_pinctrl_mux_configure(const struct npcx_alt *alts_list,
 			uint8_t alts_size, int altfunc);
 
+/**
+ * @brief Select i2c port pads of i2c controller
+ *
+ * @param controller i2c controller device
+ * @param port index for i2c port pads
+ */
+void soc_pinctrl_i2c_port_sel(int controller, int port);
```

Comment:
I still think all this should be namespaced specific to the platform, i.e. `npcx_pinctrl_` rather than `soc_pinctrl`, but since it's already this way such a change should be done separately.

Reply:
Hi: @pabigot 
All single-line statement blocks have added braces in this PR. Regarding namespaced specific to the platform, will have a PR for it.

Reply:
Submit [PR29387](https://github.com/zephyrproject-rtos/zephyr/pull/29387) for 'rename' request.